### PR TITLE
[HIU_639] | Mounica/Aditi | Verify login credentials with account service

### PR DESCRIPTION
**Pull request by imaditiagg**

### CODE REVIEW

File: src/main/java/in/org/projecteka/hiu/HiuApplication.java [hunk 1]
```diff
@@ -12,7 +13,8 @@
 @EnableConfigurationProperties({ConsentManagerServiceProperties.class,
                                 RabbitMQOptions.class,
                                 CacheMethodProperty.class,
-                                RedisOptions.class})
+                                RedisOptions.class,
+		                        AccountServiceProperties.class})
```
Comment (ganesanarun) [inline src/main/java/in/org/projecteka/hiu/HiuApplication.java]: format this please.

File: src/main/java/in/org/projecteka/hiu/clients/AccountServiceProperties.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+@ConfigurationProperties(prefix = "hiu.accountservice")
+@AllArgsConstructor
+@ConstructorBinding
+@Getter
+public class AccountServiceProperties {
+        private final boolean usingUnsecureSSL;
+        private final String url;
+        private final List<String> identifiers;
+        private final int expiryInMinutes;
+
```
Comment (ganesanarun) [inline src/main/java/in/org/projecteka/hiu/clients/AccountServiceProperties.java]: nic: remove this extra line.

File: src/main/java/in/org/projecteka/hiu/common/CMAccountServiceAuthenticator.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+import java.text.ParseException;
+
+import static java.lang.String.format;
+
+@AllArgsConstructor
+public class CMAccountServiceAuthenticator implements Authenticator {
+    private final SessionServiceClient sessionServiceClient;
+    private final ConsentManagerServiceProperties consentManagerServiceProperties;
+    private final Logger logger = LoggerFactory.getLogger(CMAccountServiceAuthenticator.class);
```
Comment (ganesanarun) [inline src/main/java/in/org/projecteka/hiu/common/CMAccountServiceAuthenticator.java]:
```suggestion
    private static final Logger logger = LoggerFactory.getLogger(CMAccountServiceAuthenticator.class);
```

**Comment by ganesanarun** [comment]
https://sonarcloud.io/project/issues?id=ProjectEKA_health-information-user&pullRequest=216&resolved=false&types=CODE_SMELL fix this as well.

File: src/main/java/in/org/projecteka/hiu/HiuApplication.java (hunk 1 above)
Comment (imaditiagg) [reply src/main/java/in/org/projecteka/hiu/HiuApplication.java]: Done

File: src/main/java/in/org/projecteka/hiu/clients/AccountServiceProperties.java (hunk 2 above)
Comment (imaditiagg) [reply src/main/java/in/org/projecteka/hiu/clients/AccountServiceProperties.java]: Done

File: src/main/java/in/org/projecteka/hiu/common/CMAccountServiceAuthenticator.java (hunk 3 above)
Comment (imaditiagg) [reply src/main/java/in/org/projecteka/hiu/common/CMAccountServiceAuthenticator.java]: Done

File: src/main/java/in/org/projecteka/hiu/clients/AccountServiceProperties.java (hunk 2 above)
Comment (imaditiagg) [reply src/main/java/in/org/projecteka/hiu/clients/AccountServiceProperties.java]: May I know what is nic ?

Comment (ganesanarun) [reply src/main/java/in/org/projecteka/hiu/clients/AccountServiceProperties.java]: #### nitpick

**Review by ganesanarun — APPROVED** [review]

**Pull request merged**